packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +84,27 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+/// An expansion panel that allows for radio-like functionality.
+///
+/// A unique identifier [value] must be assigned to each panel.
+class ExpansionPanelRadio extends ExpansionPanel {
+
+  /// A subclass of expansion panel that allows for radio functionality.
+  ///
+  /// A unique identifier [value] must be passed into the constructor. The
```

Comment:
Take out the word "identifier": it reads better as "A unique [value]".

Also, explain why it must be unique: say something like "The [value] is used to indicate which panel is currently selected."